src/components/common/Tour.js

```diff
@@ -75,8 +77,15 @@ export class Tour extends Component {
       push: PropTypes.func.isRequired,
     }).isRequired,
     t: PropTypes.func.isRequired,
+    showTours: PropTypes.bool.isRequired,
+    dispatchGetToursEnabled: PropTypes.func.isRequired,
   };
 
+  componentDidMount() {
+    const { dispatchGetToursEnabled } = this.props;
+    dispatchGetToursEnabled();
```

Comment:
Don't you want to dispatch this earlier on in the app?

Reply:
`Tour` is located in `App` which is pretty high in the hierarchy I would say. Plus this will be called only if the component is used (since it is the global component for all tours). If you think it is nevertheless better to do it higher in `App` or even in `Root` let me know.